src/App/VisibilityBar.tsx

```diff
@@ -32,13 +33,49 @@ const VisibilityBar: FC<{ isSidePanelEnabled: boolean }> = ({
     const isLogVisible = useSelector(isLogVisibleSelector);
     const autoScroll = useSelector(autoScrollSelector);
 
+    useHotKey({
+        hotKey: 'shift+m',
+        title: 'Show/Hide menu',
```

Comment:
I think calling this “menu” is confusing because it isn't a side panel, not really a menu. I know that we also call it “menu” in the bar at the bottom and we should probably talk to Ketil whether we also rename it there. At least I expect something very different when it comes to a “menu” (e.g. a menu bar).

Reply:
We just saw "menu" on the button and used that name 🤷